### ISSUE TITLE
[Enhancement] Customizable class for external links

### ISSUE THREAD

**Issue by Swader** [Sommerregen/grav-plugin-external-links#14]
It would be good if there was an option to define a custom class for the external links. The reasoning is the [langswitcher](https://github.com/getgrav/grav-plugin-langswitcher/blob/develop/templates/partials/langswitcher.html.twig#L14) forcing the external class on the language links, which then makes the language links appear external, even if it is not so.

Of course, this problem can be bypassed by extending the plugin's partial and removing the forced external class, but it's more work for the end user, who seeks simplicity.

**Comment by Sommerregen** [comment]
@Swader Didn't noticed. I will consider your idea in the next release. Instead of exposing a class name, I really drop the support for that in the stylesheet [L1](https://github.com/Sommerregen/grav-plugin-external-links/blob/master/assets/css/external_links.css#L1). Is this an option you can live with?

**Comment by Swader** [comment]
I don't see any negatives right now, so sure.

**Comment by Sommerregen** [comment]
Sorry, to say something different, but today I thought about it. A class name `external` semantic and is something really special. It says that the link will be "external" or something like that and it doesn't make sense, that it is used in **Langswitcher plugin**. Clicking on a different language does not redirect you to any external site.

TLDR: Opening an issue on [langswitcher plugin](https://github.com/getgrav/grav-plugin-langswitcher/issues), because it should be addressed there and not here...

**Comment by Sommerregen** [comment]
Will close due to inactivity and wrong repo. Feel free to open it again.
